Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/mtl_state_cache.h

```diff
@@ -359,7 +359,7 @@ struct RenderPassDepthAttachmentDesc : public RenderPassAttachmentDesc
         return !(*this == other);
     }
 
-    double clearDepth = 0;
+    double clearDepth = 1.0;
```

Comment:
Does this change the default depth clear value from 0.0 to 1.0? I don't think we can do that, that would break WebGL?

Reply:
Although I don't think it hurts since the default value of a mtl render pass depth attachment clearDepth is 1